internal/pkg/cli/storage_init.go

```diff
@@ -86,8 +99,19 @@ var attributeTypes = []string{
 	ddbBinaryType,
 }
 
+// RDS Aurora Serverless specific questions and help prompts.
+var (
+	storageInitRDSInitialDBNamePrompt = "What would you like to name the initial database in your cluster?"
+	storageInitRDSInitialDBNameHelp   = "The name of the initial database in the cluster."
+
+	storageInitRDSDBEnginePrompt = "Which database engine would you like to use?"
+	storageInitRDSDBEngineHelp   = "The database engine used in the cluster."
+)
+
 // RDS Aurora Serverless specific constants and variables.
 const (
+	rdsStorageNameDefault = "aurora-cluster"
+
 	engineTypeMySQL      = "MySQL"
 	engineTypePostgreSQL = "PostgreSQL"
```

Comment:
Can we re-use the constants defined in the `addon` pkg?

Reply:
I intended to conceptually separate what's rendered to user from what's the underlying type. So these two variables are the user-level presentation of the engines while the ones in `addon` package is how our internal packages talk to each other. In case we want to render different keywords (such as MySQL 1.6), it should be be only an UI change (so only a change in `cli/storage_init.go`), but not an `addon` change. 

However since they have the same value at this moment, I think it also works if we use the same variable and reference across package 👀